appveyor.yml

```diff
@@ -78,6 +78,8 @@
               BOND_BOOST: "1.63.0"
               BOND_CMAKE_FLAGS: "-DBOND_SKIP_CORE_TESTS=TRUE;-DBOND_SKIP_GBC_TESTS=TRUE"
     install:
+        - sc config wuauserv start= auto
```

Comment:
Interesting... How did you figure out that the Windows Update service was causing problems?

Reply:
I got error message "ERROR: Update KB3035131 installation failed (exit code 1058)." in the build log,
Issue was addressed here cocur/slugify#193

Reply:
Weird... Thanks for the details.